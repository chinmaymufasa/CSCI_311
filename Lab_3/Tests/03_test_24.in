2
48
5
2
54
5
2
84
5
2
178
5
2
198
5
1
5
-1