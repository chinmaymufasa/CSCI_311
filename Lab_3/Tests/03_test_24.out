48 
48 54 
48 54 84 
48 54 84 178 
48 54 84 178 198 
Not found
